doc/rtd/development/dir_layout.rst

```diff
@@ -3,85 +3,62 @@
 Directory layout
 ****************
 
-``Cloud-init``'s directory structure is somewhat different from a regular
-application: ::
-
-  /var/lib/cloud/
-      - data/
-         - instance-id
-         - previous-instance-id
-         - datasource
-         - previous-datasource
-         - previous-hostname
-      - handlers/
-      - instance
-      - instances/
-          i-00000XYZ/
-            - boot-finished
-            - cloud-config.txt
-            - datasource
-            - handlers/
-            - obj.pkl
-            - scripts/
-            - sem/
-            - user-data.txt
-            - user-data.txt.i
-      - scripts/
-         - per-boot/
-         - per-instance/
-         - per-once/
-      - seed/
-      - sem/
-
 ``/var/lib/cloud``
+==================
```

Comment:
In dropping the top directory tree, we now have slightly confusing rendering of the subdirectories as headings equal to the font and weight of the parent /var/lib/cloud. Let's make them subheadings so they are deemphasized.

Reply:
I've changed this instead to ellipsis to represent that the elements are subdirectories until /var/lib/cloud. The result looks like this:

![image](https://github.com/canonical/cloud-init/assets/14128326/bfe9cddb-ab17-4e84-8554-ff922a4d7f7c)